scripts/gen-env-files.sh

```diff
@@ -72,11 +72,28 @@ else
 echo \
 "ASPNETCORE_ENVIRONMENT=Local
 ASPNETCORE_URLS=http://*:5000
+
 TZ=America/Los_Angeles
+
 ConnectionStrings__PIMS=Server=localhost,5433;uid=$varApiDb;Password=$passvar;Database=pims
 DB_PASSWORD=$passvar
+
 Keycloak__Secret=
-Keycloak__ServiceAccount__Secret=" >> ./backend/api/.env
+Keycloak__ServiceAccount__Secret=
+
+Ltsa__MyLtsaUsername=
+Ltsa__MyLtsaUserPassword=
+Ltsa__IntegratorUsername=
+Ltsa__IntegratorPassword=
```

Comment:
@devinleighsmith These ones were not there but looked necessary given we are retrieving LTSA data. Any concerns?

Reply:
Having them here is a good idea, as long as none of them are set.